src/usr/local/www/services_ntpd.php

```diff
@@ -210,6 +203,13 @@ function build_interface_list() {
 
 $section = new Form_Section('NTP Server Configuration');
 
+$section->addInput(new Form_Select(
+	'enable',
+	'Enable NTP Server',
+	$pconfig['enable'],
+	array('enabled' => 'Enabled', 'off' => 'Disabled')
+));
```

Comment:
The Form_Checkbox() class should be used here. There are plenty of examples throughout the codebase.

e.g.: https://github.com/pfsense/pfsense/blob/master/src/usr/local/www/services_snmp.php#L213